recipes/onnx/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "onnx" %}
+{% set version = "0.2" %}
+{% set sha256 = "63dde5e5f3aa88a8993c65ca35960bac06ed2233001924db28a611b7c40c6254" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - protobuf
+    - numpy
+    - pytest-runner
+  run:
+    - python
+    - protobuf
+    - numpy
+
+about:
+  home: https://github.com/onnx/onnx/
+  license: BSD
```

Comment:
https://github.com/onnx/onnx/blob/master/LICENSE says MIT.

You should also add a `license_file` (legally required for us under the license). I see you're a committer for `onnx`, so you can just add `include LICENSE` to `MANIFEST.in` and it'll be included in the next release. If you'd rather get it in conda-forge before a new release comes out, you can [package it manually](https://conda-forge.org/docs/meta.html#packaging-the-license-manually).